include/eve/module/core/regular/impl/add.hpp

```diff
@@ -32,20 +29,20 @@ add_(EVE_SUPPORTS(cpu_),
 // N parameters
 //================================================================================================
 template<decorator D, value T0, value... Ts>
```

Comment:
I think the `<concept/value.hpp>` is still needed.

Reply:
value is included by common_value